extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcTenantConfig.java

```diff
@@ -831,6 +831,13 @@ enum ResponseMode {
         @ConfigDocDefault("true when UserInfo bean is injected, false otherwise")
         Optional<Boolean> userInfoRequired();
 
+        /**
```

Comment:
@michalvavrik What I was saying, I doubt we should this new shortcut here alongside the actual ConfigMapping property just above it

Reply:
But we mostly rely on the `ConfigMapping` interfaces. Once setters and fields are removed it won't matter because from users POV they will be interchangeable, but for now, I don't know better place to add it. Let's not add it for now then.

Reply:
I left it here by a mistake.